src/library_vr.js

```diff
@@ -32,11 +33,20 @@ var LibraryWebVR = {
       navigator.getVRDisplays().then(function(displays) {
         WebVR.ready = true;
         WebVR.displays = displays;
+        WebVR.displayNames = new Array(displays.length);
+        callback();
       });
 
       return 1;
     },
 
+    deinit: function() {
+      WebVR.displayNames.forEach(function(name) {
+          _free(name);
```

Comment:
style note, should be 2 spaces

Reply:
Thanks! 👍 I should really configure my editor correctly.